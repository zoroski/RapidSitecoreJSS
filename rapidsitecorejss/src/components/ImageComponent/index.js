import React from 'react';
import { Image } from '@sitecore-jss/sitecore-jss-react';

const ImageComponent = (props) => (
  <div className="banners row my-2">
    <Image field={props.fields.imageFields} className="col-12 img-fluid banner" />
  </div>
);

export default ImageComponent;
