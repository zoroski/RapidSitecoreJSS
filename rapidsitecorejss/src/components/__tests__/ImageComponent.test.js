import React from 'react';
import ImageComponent from '../ImageComponent/index';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
      const tree = renderer
        .create( < ImageComponent fields = {
            {
              title: {
                value: 'Title',
              },
              imageFields: {
                value: {
                  src: '/media/image.png'
                },
              },
            }
          }
          />)
          .toJSON(); expect(tree).toMatchSnapshot();
        });