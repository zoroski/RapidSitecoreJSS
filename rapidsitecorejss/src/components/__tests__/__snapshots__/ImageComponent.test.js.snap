// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  class="banners row my-2"
>
  <img
    class="col-12 img-fluid banner"
    src="/media/image.png"
  />
</div>
`;
